apps/client/src/app/components/user-account-access/create-or-update-access-dialog/create-or-update-access-dialog.html

```diff
@@ -22,9 +22,21 @@ <h1 i18n mat-dialog-title>Grant access</h1>
         <mat-label i18n>Type</mat-label>
         <mat-select formControlName="type">
           <mat-option i18n value="PUBLIC">Public</mat-option>
+          <mat-option i18n value="PRIVATE">Private</mat-option>
         </mat-select>
       </mat-form-field>
     </div>
+    <div *ngIf="accessForm.controls['type'].value === 'PRIVATE' ">
```

Comment:
You can use the [new control flow](https://angular.dev/guide/templates/control-flow#if-block-conditionals) of Angular:
```
@if (accessForm.controls['type'].value === 'PRIVATE') {
  <div>
    ...
  </div>
}
```